xds/src/test/java/io/grpc/xds/internal/rbac/engine/GrpcAuthorizationEngineTest.java

```diff
@@ -223,7 +229,8 @@ public void headerMatcher_aliasAuthorityAndHost() {
     PolicyMatcher policyMatcher = new PolicyMatcher(POLICY_NAME, permission, principal);
     GrpcAuthorizationEngine engine = new GrpcAuthorizationEngine(
         new AuthConfig(Collections.singletonList(policyMatcher), Action.ALLOW));
-    AuthDecision decision = engine.evaluate(metadata("authority", "google"), serverCall);
+    when(serverCall.getAuthority()).thenReturn("google");
```

Comment:
Use "google.com" or "example.com" or similar to make it clear this is a domain name?